packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -627,7 +635,11 @@ describeNoCompat("Garbage collection of blobs", (getTestObjectProvider) => {
             // Attach the container after the blob is uploaded.
             await container.attach(provider.driver.createCreateNewRequest(provider.documentId));
 
-            // Upload the same blob. This will get de-duped and we will get back a handle with the stoageId instead of
+            // Make sure we are connected or we may get a local ID handle
+            defaultDataStore._root.set("make container connect in", "write mode");
+            await ensureContainerConnectedWriteMode(container as Container);
```

Comment:
Just waiting for connected should be enough here right. You don't need to wait for write mode explicitly.